src/stores/notifications/RoomNotificationState.ts

```diff
@@ -51,6 +52,57 @@ export class RoomNotificationState extends NotificationState implements IDestroy
         cli.removeListener(ClientEvent.AccountData, this.handleAccountDataUpdate);
     }
 
+    /**
+     * True if the notification is an invitation notification.
+     * Invite notifications are a special case of highlight notifications
+     */
+    public get isInvitation(): boolean {
+        if (this.knocked) return false;
+
+        return this.level === NotificationLevel.Highlight && this.symbol === "!" && this.count === 1;
```

Comment:
This feels mixed up: at the moment this class is operating just at the level of what should be displayed on the screen, but this is going back to the semantics of what the notification is which is why it's having to reverse-engineer it from the symbol. If the invite specifically being an invite is a thing this class cares about, it should probably store that explicitly.

Reply:
You're right, I added another parameter to the ``

See in [`578e622` (#29564)](https://github.com/element-hq/element-web/pull/29564/commits/578e622de2d43b37982c8837c6457cf532334593)

Updated tests in [`95e18c5` (#29564)](https://github.com/element-hq/element-web/pull/29564/commits/95e18c54f5ead109f62bb7505a3a3c0a8b9faf86) and [`6914324` (#29564)](https://github.com/element-hq/element-web/pull/29564/commits/69143244264725524e30a497041cc0dd95774996)